src/app/main-page/main-page.component.css

```diff
@@ -5,3 +5,17 @@
 .buttonPage {
   height: 80vh;
 }
+
+.scanRegion {
```

Comment:
This could be a generic, reusable glass pane style. Might be worth making it as such and we can move it somewhere more sharable later if we need it.

Reply:
Having said that there's possibly no point due to having to handle mouse events somewhere else.... Maybe a whole component with behaviour would be better....hmmmmm